StabQuest/GameStates/LevelUpState.cs

```diff
@@ -0,0 +1,285 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Graphics;
+using StabQuest.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Net.Mime.MediaTypeNames;
+using static StabQuest.Helpers.DiceHelper;
+
+namespace StabQuest.GameStates
+{
+    public class LevelUpState : GameState
+    {
+        private SpriteFont _font;
+        private Character _character;
+        private int[] _dirtyStatArray;
+        private List<GameComponent> _components;
+        private int _unspent;
+        public LevelUpState(ContentManager content, GraphicsDevice graphicsDevice, Game1 game, Character character) : base(content, graphicsDevice, game)
+        {
+            var boxTexture = content.Load<Texture2D>("Images/box");
+            var buttonTexture = content.Load<Texture2D>("Images/button");
+            _font = content.Load<SpriteFont>("MyFont");
+            _character = character;
+
+            ResetNewStats();
+
+            var str_up = new Button(new Vector2(50, 100), boxTexture, _font)
```

Comment:
Maybe extract this into a private InitializeUI method?

Reply:
Thought about it, probably correct

Reply:
Fixed